code/obj/item/plants.dm

```diff
@@ -421,11 +421,14 @@ ABSTRACT_TYPE(/obj/item/plant/herb)
 	Crossed(atom/movable/AM as mob|obj)
 		var/mob/M = AM
 		if(iswerewolf(M))
-			M.changeStatus("weakened", 3 SECONDS)
-			M.force_laydown_standup()
-			M.TakeDamage("All", 0, 5, 0, DAMAGE_BURN)
-			M.visible_message("<span class='alert'>The [M] steps too close to [src] and falls down!</span>")
-			return
+			var/stun_duration
+			if (ON_COOLDOWN(M, "aconite_stun", stun_duration))
```

Comment:
since stun applied is shorter than 3 seconds it seems to give ww a pretty large invincibility frame to run over other piles of aconite that can be ahead, unless its fine with you?